### DISCUSSION TITLE
多开CHIA 导至有一个爆块，接收地址是以前地址，还能找回币吗，原始注册助记词还在地址钱包

### DISCUSSION THREAD

**Discussion by sky7282** [Chia-Network/chia-blockchain#7655] [Support]
请问一下，爆块了，币找不到，查了一下在最先的那个助记词注册的地址里面，但是后面重新登记时接收地址都是新的，不一样了。怎么找回币，急需高手支持一下，谢谢。

**Comment by keliew** [comment]
Same mnemonics will show the same balance in your wallet regardless of the wallet address. If mnemonics are difference, then no.
